infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/AddNodeWorkflow.java

```diff
@@ -141,9 +142,15 @@ protected void stateTransfer(Set<String> endpoints, CorfuRuntime runtime,
         for (String endpoint : endpoints) {
             // TrimMark is the first address present on the log unit server.
             // Perform the prefix trim on the preceding address = (trimMark - 1).
+            // Since the LU will reject trim decisions made from older epochs, we
+            // need to adjust the new trim mark to have the new layout's epoch.
+            Token prefixToken = new Token(newLayout.getEpoch(), trimMark - 1);
+            CFUtils.getUninterruptibly(runtime.getLayoutView().getRuntimeLayout(newLayout)
+                    .getBaseClient(endpoint).setRemoteEpoch(newLayout.getEpoch()));
```

Comment:
How do writes bypass the server router epoch check on master?

Reply:
Bootstrap was setting the layout, which was setting the server epoch. 
Should remove this explicit setRemoteEpoch and let the bootstrap path seal the servers.